Feature#27 add unit test for all models

mariormt17 opened the pull request:
<!--- PR Title: What changed/implement in this PR -->
<!--- Be specific, don't write a generic title please --> 
 
## Description: Why did you change/implement that?
<!-- Describe your changes in detail -->
This PR is to evaluate the unit tests of the models in the project.

## Testing: Help me how can I test or look at the changes?
<!-- Please describe in detail how you tested your changes or use screenshots -->

Commit:
seve config schema

Commit:
Merge branch 'master' of github.com:Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08

Commit:
Merge branch 'master' of github.com:Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08

Commit:
Merge branch 'master' of github.com:Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08

Commit:
Merge branch 'master' of github.com:Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08

Commit:
add gem for test

add gem factory_girl_rails for tests

mariormt17 committed:
Merge branch 'master' of https://github.com/Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08

mariormt17 committed:
Merge branch 'feature#27-add-unit-test-for-all-models' of https://github.com/Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08 into feature#27-add-unit-test-for-all-models

mariormt17 committed:
Employee and Attendance tests added

Commit:
modification spec_helper

this file is updete when was reinstalled rspec

Commit:
Merge branch 'feature#27-add-unit-test-for-all-models' of github.com:Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08 into feature#27-add-unit-test-for-all-models

Commit:
Modification address file

in this commit was modificated the address for file rails helper required for the unit test

Commit:
add test company model

add two test for validated the attributes name and address and one test for create a new company

ramasoftdev committed:
Adding companies_specs type request and configurated session admin

ArielMD reviewed (dismissed):
I tested this, and it works

ramasoftdev committed:
Request test for welcome maded

Commit:
modification gem axlsx

change the call to gem axlsx for caxlsx

Commit:
update tdd on company

modification in structure of file company_spec for best develop of the test and add new validation

mariormt17 committed:
Tests updated on Attendance and Employee spec

mariormt17 committed:
Fix failed test in employee spec

Commit:
update tdd in admin

add the new test for admin_spec with unit test for validation of attributes in this model

ramasoftdev committed:
Companies`s request specs finished and modified companies controllers

ramasoftdev committed:
Adding checkin reuqest spec

ramasoftdev committed:
Adding resports`s specs

ramasoftdev committed:
Adding reuqest spec for employees, requests specs finished

mariormt17 commented:
Now the employee tests works fine

Commit:
Merge branch 'feature#27-devise-login-test' into feature#27-add-unit-test-for-all-models

Commit:
modification rails helper

update address file on rails helper for test in models

ramasoftdev committed:
Solving file not found error in tests

FranckZnt19 committed:
Merge branch 'master' into feature#27-add-unit-test-for-all-models
